internal/service/hub/handler/nta/node_register.go

```diff
@@ -363,6 +375,41 @@ func (n *NTA) checkSignature(_ context.Context, address common.Address, message
 	return nil
 }
 
+// checkEndpoint checks if the endpoint is available and contains the node's address.
+func (n *NTA) checkEndpoint(ctx context.Context, endpoint string, address common.Address) error {
+	httpRequest, err := http.NewRequestWithContext(ctx, http.MethodGet, fmt.Sprintf("http://%s:81", endpoint), nil)
```

Comment:
1. should import the HTTP provider to replace it with the fetch method.
2. the node endpoint has already had a schema added during registration. 
https://github.com/RSS3-Network/Global-Indexer/blob/main/internal/service/hub/handler/nta/node_register.go#L120